dotcom-rendering/src/web/components/Card/components/CardWrapper.tsx

```diff
@@ -9,13 +9,14 @@ type Props = {
 	format: ArticleFormat;
 	containerPalette?: DCRContainerPalette;
 	containerType?: DCRContainerType;
-	transparent?: boolean;
+	isDynamo?: boolean;
```

Comment:
Explain what a Dynamo is, and only accept the prop if explicitly `true`.

```suggestion
	/** The first card in a dynamic package is ”Dynamo” and gets special styling */
	isDynamo?: true;
```

Reply:
Love this. Implemented 👍